policy/monocamMapper.paf

```diff
@@ -475,4 +475,17 @@ datasets: {
         tables: raw
         tables: raw_visit
     }
+    apPipe_config: {
+        template:      "config/apPipe.py"
+        python:        "lsst.ap.pipe.ApPipeConfig"
+        persistable:      "Config"
+        storage:    "ConfigStorage"
+    }
+    apPipe_metadata: {
+        template:      "apPipe_metadata/v%(visit)d_f%(filter)s.boost"
+        python:        "lsst.daf.base.PropertySet"
+        persistable:        "PropertySet"
+        storage:    "BoostStorage"
+        tables:        raw
+    }
```

Comment:
Other than being in a .paf file instead of a .yaml file this seems to match the contents of pipe_base. Why can't you remove this change and letting pipe_base handle it (I assume the answer will be the same for obs_comCam).